test/utils.test.js

```diff
@@ -1548,25 +1548,45 @@ describe('getActionUrls', () => {
 
   test('local dev true', () => {
     const expected = {
-      'sample-app-1.0.0/action': 'https://adobeioruntime.net/api/v1/web/fake_ns/sample-app-1.0.0/action',
-      'sample-app-1.0.0/action-sequence': 'https://adobeioruntime.net/api/v1/web/fake_ns/sample-app-1.0.0/action-sequence',
-      'sample-app-1.0.0/action-zip': 'https://adobeioruntime.net/api/v1/web/fake_ns/sample-app-1.0.0/action-zip'
+      'sample-app-1.0.0-action': 'https://adobeioruntime.net/api/v1/web/fake_ns/sample-app-1.0.0/action',
+      'sample-app-1.0.0-action-sequence': 'https://adobeioruntime.net/api/v1/web/fake_ns/sample-app-1.0.0/action-sequence',
+      'sample-app-1.0.0-action-zip': 'https://adobeioruntime.net/api/v1/web/fake_ns/sample-app-1.0.0/action-zip'
     }
     const result = utils.getActionUrls(config, config.actions.devRemote, true)
     expect(result).toEqual(expect.objectContaining(expected))
   })
 
   test('web false', () => {
     const expected = {
-      'sample-app-1.0.0/action': 'https://adobeioruntime.net/api/v1/fake_ns/sample-app-1.0.0/action',
-      'sample-app-1.0.0/action-sequence': 'https://adobeioruntime.net/api/v1/fake_ns/sample-app-1.0.0/action-sequence',
-      'sample-app-1.0.0/action-zip': 'https://adobeioruntime.net/api/v1/web/fake_ns/sample-app-1.0.0/action-zip'
+      'sample-app-1.0.0-action': 'https://adobeioruntime.net/api/v1/fake_ns/sample-app-1.0.0/action',
+      'sample-app-1.0.0-action-sequence': 'https://adobeioruntime.net/api/v1/fake_ns/sample-app-1.0.0/action-sequence',
+      'sample-app-1.0.0-action-zip': 'https://adobeioruntime.net/api/v1/web/fake_ns/sample-app-1.0.0/action-zip'
     }
     config.manifest.full.packages.__APP_PACKAGE__.actions.action.web = 'no'
     config.manifest.full.packages.__APP_PACKAGE__.sequences['action-sequence'].web = false
     const result = utils.getActionUrls(config, config.actions.devRemote, true)
     expect(result).toEqual(expect.objectContaining(expected))
   })
+
+  test('multiple packages with includeWithoutPkgName = true', () => {
```

Comment:
also add test for `multiple packages includeWithoutPackage = false` (default param value for getActionUrls )